### ISSUE TITLE
flag provided but not defined: -output-version

### ISSUE THREAD

**Issue by nancheal** [CycloneDX/cyclonedx-gomod#207]
with this command
cyclonedx-gomod mod -output bom.xml -output-version 1.2 /test
i get this error
is -output-version flag available?

**Comment by nscuro** [comment]
Hey @nancheal, sorry for the confusion. 

The `-output-version` feature has been merged to `main`, but we didn't create a release yet. If you want to test or use this functionality already, you can install the "snapshot" version using this command:

```shell
go install github.com/CycloneDX/cyclonedx-gomod/cmd/cyclonedx-gomod@main
```

**Comment by nscuro** [comment]
This is now available in v1.4.0: https://github.com/CycloneDX/cyclonedx-gomod/releases/tag/v1.4.0
